tests/lib/openstack_query/managers/test_query_manager.py

```diff
@@ -41,8 +41,11 @@ def setUp(self) -> None:
 
         self.query = MagicMock()
         self.prop_cls = MagicMock()
+        self.cloud_account = MagicMock()
+        self.cloud_account.name = "test_account"
+
         self.instance = QueryManager(
-            cloud_account="test_account", query=self.query, prop_cls=self.prop_cls
+            cloud_account=self.cloud_account, query=self.query, prop_cls=self.prop_cls
```

Comment:
We should just use the mock and not set a string, that way we can check it calls the right thing:

```python
self.run.assert_called_with(self.cloud_account.name.lower.return_value)
```
This makes it clear what we're doing to our enum, rather than have a string pop out